### PR TITLE
fix: add 'in_trash' field to Notion block, database, and page models

### PR DESCRIPTION
**Pull request by hardchor**
Fixes https://github.com/Unstructured-IO/unstructured-ingest/issues/56

### CODE REVIEW

**Comment by rbiseck3** [comment]
Can this also get an update to the changelog? That'll prevent the CI linters from passing.

**Comment by rbiseck3** [comment]
Sorry, I think the changelog needs to be updated one more time to pull in changes that had since been merged in.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,224 +1,228 @@
...
-* **Migrate Outlook Source Connector to V2**
-* **Support for Databricks Volumes source connector**
+- **Support pinecone namespace on upload**
+- **Migrate Outlook Source Connector to V2**
+- **Support for Databricks Volumes source connector**
 
 ### Fixes
 
-* **Update Sharepoint Creds and Expected docs**
+- **Update Sharepoint Creds and Expected docs**
```
Comment (rbiseck3) [inline CHANGELOG.md:17]: Looks like something updated all of these, can we limit the changes to the changelog in this PR?

**Comment by potter-potter** [comment]
@hardchor Thanks for this. I'll just put it in a new PR since we have to update some of the ingest tests, etc. Will just be easier. Thanks so much for working on this!

See https://github.com/Unstructured-IO/unstructured-ingest/pull/171

**Comment by rbiseck3** [comment]
Closing in favor of new PR: https://github.com/Unstructured-IO/unstructured-ingest/pull/171

**Pull request closed without merging**